po/zh.po

```diff
@@ -1919,120 +1877,114 @@ msgstr "错误：无法将种子标记为已读: %s"
 
 #: rss/parser.cpp:282
-#, fuzzy
 msgid "could not parse file"
-msgstr "错误：无法分析过滤器（filter）命令"
+msgstr "错误：无法解析文件"
 
 #: rss/parser.cpp:307
 msgid "no RSS version"
-msgstr ""
+msgstr "没有 RSS 版本"
 
 #: rss/parser.cpp:323
-#, fuzzy
 msgid "invalid RSS version"
-msgstr "无效位置！"
+msgstr "无效的 RSS 版本"
 
 #: rss/parser.cpp:344 rss/parser.cpp:355
-#, fuzzy
 msgid "invalid Atom version"
-msgstr "无效位置！"
+msgstr "无效的 Atom 版本"
 
 #: rss/parser.cpp:360
 msgid "no Atom version"
-msgstr ""
+msgstr "没有 Atom 版本"
 
 #: rss/rss09xparser.cpp:32
 msgid "no RSS channel found"
-msgstr ""
+msgstr "没有找到 RSS 频道"
 
 #: rss/rssparserfactory.cpp:31
 msgid "unsupported feed format"
-msgstr ""
+msgstr "不支持的种子格式"
 
 #: rust/libnewsboat/src/cliargsparser.rs:215
 msgid "%s: %s: invalid loglevel value"
-msgstr ""
+msgstr "%s: %s: 无效的日志等级"
 
 #: rust/libnewsboat/src/configpaths.rs:85
-#, fuzzy
 msgid ""
 "Fatal error: couldn't determine home directory!\n"
 "Please set the HOME environment variable or add a valid user for UID %u!"
-msgstr "请设置主目录的环境变量，或者添加一个有效的用户其UID为 %u!"
+msgstr "严重错误：无法确定家目录！\n"
+"请设置主目录的环境变量，或者添加一个有效的用户其UID为 %u!"
 
 #: rust/libnewsboat/src/configpaths.rs:146
 msgid "Migrating configs and data from Newsbeuter's XDG dirs..."
-msgstr ""
+msgstr "从 Newsbeuter 的 XDG 目录迁移配置和数据..."
 
 #: rust/libnewsboat/src/configpaths.rs:202
 msgid "Migrating configs and data from ~/.newsbeuter/..."
-msgstr ""
+msgstr "从 ~/.newsbeuter/ 迁移配置和数据..."
 
 #: rust/libnewsboat/src/configpaths.rs:214
-#, fuzzy
 msgid "Aborting migration because mkdir on `%s' failed: %s"
-msgstr "错误:打开缓存文件`%s' 失败:%s"
+msgstr "迁移过程中断，在`%s'处建立目录失败：%s"
 
 #: rust/libnewsboat/src/filterparser.rs:129
-#, fuzzy
 msgid "attribute name"
-msgstr "属性没有发现"
+msgstr "属性名称"
 
 #. Don't translate "between" -- it's a keyword, not an English word.
 #: rust/libnewsboat/src/filterparser.rs:131
 msgid "one of: =~, ==, =, !~, !=, <=, >=, <, >, between, #, !#"
-msgstr ""
+msgstr "其中之一：=~、==、=、!~、!=、<=、>=、<、>、between、#、!#"
 
 #. The options ("quoted string" etc.) are not keywords, so please translate them.
 #: rust/libnewsboat/src/filterparser.rs:133
 msgid "one of: quoted string, range, number"
-msgstr ""
+msgstr "其中之一：quoted string、range、number"
 
 #. The first %s is an integer offset at which trailing characters start, the
 #. second %s is the tail itself.
 #: rust/libnewsboat/src/filterparser.rs:426
 msgid "Parse error: trailing characters after position %s: %s"
-msgstr ""
+msgstr "解析错误：在位置 %s 之后有尾随字符：%s"
 
 #. The first %s is a zero-based offset into the string, the second %s is the
 #. description of what the program expected at that point.
 #: rust/libnewsboat/src/filterparser.rs:433
 msgid "Parse error at position %s: expected %s"
-msgstr ""
+msgstr "位置 %s 处解析错误：期望得到 %s"
 
 #: rust/libnewsboat/src/filterparser.rs:437
 msgid "Internal parse error"
-msgstr ""
+msgstr "内部解析错误"
 
 #: rust/libnewsboat/src/fslock.rs:57
-#, fuzzy
 msgid "Failed to open lock file '%s': %s"
-msgstr "当调用`%s'的时候出错: %s"
+msgstr "打开锁文件 '%s' 失败: %s"
```

Comment:
In the translations above, ``` `%s' ``` was not surrounded by spaces. Now with `'%s'` there are spaces placed around it. This looks inconsistent to me (and yes, Newsboat is inconsistent with the used types of quotes, too). We can postpone the decision and you make up your mind which way would be best for Chinese and then fix it in a future merge request. Up to you. :-)

Reply:
It's a habit to have spaces between English letters and Chinese words for readability. I focused on the parts with `fuzzy` tags this time so I failed to concentrate on styles. Besides, I noticed the previous translations that I didn't change are also inconsistent, so maybe it's better to create a PR focusing on styles later?